internal/gensample/gensample.go

```diff
@@ -233,6 +233,7 @@ func (g *generator) commit(gofmt bool, year int) ([]byte, error) {
 
 	var file bytes.Buffer
 	fmt.Fprintf(&file, license.Apache, year)
+	fmt.Fprintf(&file, "// +build sample\n\n")
```

Comment:
By convention, build constraints are typically placed at the top of the file on the first line with a new line following. I know the docs don't say this specifically, but it is the common convention and what you will find in google-cloud-go

Reply:
Got it. Will update it shortly.